samples/plugins/impala_jdbc_delegationUID/dialect.tdd

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<dialect name='ImpalaSampleDialect'
+         class='myimpala'
+         base='PostgreSQL90Dialect'
+         version='18.1'>
+   <function-map>
+    <function name='FLOAT' group='cast' return-type='real'>
+      <formula>CAST(((EXTRACT(EPOCH FROM %1) / ( 60 * 60 * 24)) + (365 * 70 + 17)) AS BIGINT)</formula>
+      <argument type='date' />
+    </function>
+   </function-map>
+</dialect>
```

Comment:
This is the same as the postgres dialect, which is incompatible with the Impala dialect. You can use the impala dialect as a base I think, we also have a hive example tdd if needed.

Reply:
I modified base dialect to Impala23Dialect however I cannot test this since I don't have a test environment after losing access. I tried to create a test environment on public AWS but the cloudera [quickstart](https://docs.aws.amazon.com/quickstart/latest/cloudera/welcome.html) cloudformation scripts failed to create CFN stack at first. However when I setup my own VPC, the scripts did pass and I was able to create a director instance. However the cfn scripts did not download and setup other config needed for creating nodes and I figured it would require a lot of manual labor to setup properly. I suggest we should test it in internal test env with the new base dialect if this is one of the release criteria.

Reply:
It doesn't have to be a fully-fledged connector to fulfill the role of demonstrating database impersonation. However, some people may assume that it is an exmaple of an Impala connector, and has the same functionality as our built-in one. They will think can make an impala-based connector using this as a starting point. 

If it's possible to use postgres instead to demonstrate DelegationUID, then that might be a better option. It may also be a good idea to rename the sample to something like "database_impersonation_example", so that people know that the purpose of the connector is to demonstrate database impersonation and not how to make an impala-based connector.

Reply:
I am afraid, I think postgres does not support this kind of database impersonation using delegation UID or at-least I do not know of that right now. Postgres does support kerberos delegation, however these changes are not about that and we plan to support those later.

I will rename the sample plugin